justfile

```diff
@@ -73,3 +74,10 @@ docs-serve: docs-build-dependent
 
 docs-build: docs-build-dependent
     mkdocs build 
+
+markdownlint:
+    docker run \
+    -v $PWD:/workdir ghcr.io/igorshubovych/markdownlint-cli:latest \
+    --config .markdownlint.yml \
+    --disable MD013 MD040 -- \
```

Comment:
the advantage to disable these rules here is that we can leave them enabled in `.markdownlint.yml`, so that you still see the warnings in your IDE.

MD013: line too long
MD040: fenced code blocks need a language